File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.py
class Solution:
    def minimumLength(self, s: str) -> int:
        n=len(s)
        se=set(s)
        for i in se:
            k=s.count(i)
            n-=k
            if k%2==0:
                n+=2
            else:
                n+=1
        return n
            
            


        

File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.java
class Solution {
    public int minimumLength(String s) {
        int n=s.length();
        int[] hash=new int[26];
        for(int i=0;i<n;i++)
        {
            char ch=s.charAt(i);
            hash[ch-'a']++;
        }
        for(int i=0;i<26;i++)
        {
            if(hash[i]>0)
            {
                n=n-hash[i];
                if(hash[i]%2==0)
                    n+=2;
                else
                    n++;
            }
        }
        return n;
        
    }
}